### PR TITLE
1019 LinuxC create the stream encryption guide for ma

### PR DESCRIPTION
**Pull request by Pankajg123**
Issue:
https://github.com/AgoraIO/Docs-Source/issues/1019

Changes made for IOT media stream encryption guide for LinuxC..

There's one issue I found :

After launching the app with Linux terminal (by compiling the updated code with enabling encryption), if the same channel is  joined by another user (Agora web demo console user, which is Unauthorised one  as it desn't have encryption key and salt to connect with first user), then the compiled app got an warning 
`[WRN] Error 120 is captured. Error msg "audio:decrypt error"`
Which is perfectly fine. But it suddenly  stop , which should not be the case.
```
root@DESKTOP-H5KUOIH:/home/varun_gupta_py/Agora/agora_rtsa_sdk/example/out/x86_64# ./hello_rtsa --app-id "5db9f968c5214f009c7e8954c2ae82b5" --channel-id "encryption_test3" --token "007eJxTYPgceflxmqLUR1PPCJnq1M9qgdUbplqp/2B2irpmclNQfI0Cg2lKkmWapZlFsqmRoUmagYFlsnmqhaWpSbJRYqqFUZLpTauvyQ2BjAy7+hazMDJAIIgvwJCal1xUWVCSmZ8XX5JaXGLMwAAAm24j0w=="
[INF] Welcome to RTSA SDK v1.9.0
[INF] File parser found: h264
[INF] File parser found: pcm
[INF] [conn-1] Join the channel encryption_test3 successfully, uid 98412917 elapsed 5150 ms
[INF] [conn-1] Bandwidth change detected. Please adjust encoder bitrate to 499 kbps
[INF] [conn-1] Bandwidth change detected. Please adjust encoder bitrate to 417 kbps
[INF] [conn-1] Bandwidth change detected. Please adjust encoder bitrate to 432 kbps
[INF] [conn-1] Bandwidth change detected. Please adjust encoder bitrate to 636 kbps
[INF] [conn-1] Bandwidth change detected. Please adjust encoder bitrate to 654 kbps
[INF] [conn-1] Bandwidth change detected. Please adjust encoder bitrate to 672 kbps
[INF] File parser has reached the end of file. Now rewind ...
[WRN] [conn-1] Frame loss detected. Please notify the encoder to generate key frame immediately
[WRN] [conn-1] Frame loss detected. Please notify the encoder to generate key frame immediately
[WRN] [conn-1] Frame loss detected. Please notify the encoder to generate key frame immediately
[INF] [conn-1] Remote user "3421177439" has joined the channel, elapsed 13340 ms
[WRN] Error 120 is captured. Error msg "audio:decrypt error"
root@DESKTOP-H5KUOIH:
```

### CODE REVIEW

**Review by billy-the-fish — APPROVED** [review]
LGTM.

File: shared/iot/develop/_ensure-channel-quality.mdx [hunk 1]
```diff
@@ -44,7 +46,10 @@ To provide the best audio and video quality in your <Vpl k="CLIENT"/>:
 The following figure shows the workflow you need to implement to ensure channel quality in your <Vpl k="CLIENT"/>:
 
 ![Ensure Channel Quality](/images/iot/iot-channel-quality.svg)
-
+<PlatformWrapper platform="linux-c">
+**This page is coming soon for Linux c.**
+</PlatformWrapper>
+<PlatformWrapper notAllowed="linux-c">
 ## Prerequisites
```
Comment (atovpeko) [inline shared/iot/develop/_ensure-channel-quality.mdx:53]: Two questions. 
1. Does it make sense to say that this page is coming soon for Linux only after the whole intro and diagram?
2. The variable for "app" or whatever it is does not show in text on the Linux page:
Screenshot
Same for other doc pages. For example, see "Secure channel encryption"

File: shared/iot/develop/encrypt-media-streams/project-implementation/linux-c.mdx [hunk 2]
```diff
@@ -1,4 +1,102 @@
 <PlatformWrapper platform="linux-c">
```
Comment (atovpeko) [inline shared/iot/develop/encrypt-media-streams/project-implementation/linux-c.mdx:2]: The whole "Encrypt media streams" document does not show in the local build. Is this supposed to be so?

File: shared/iot/develop/_ensure-channel-quality.mdx (hunk 1 above)
Comment (Pankajg123) [reply shared/iot/develop/_ensure-channel-quality.mdx:53]: 1. Inro & diagam is there. rest I will covered with drafting page for "Channel Quality" in different PR.
2. It has been fixed in another PR.

**Pull request merged**